package.json

```diff
@@ -2,18 +2,22 @@
   "name": "incremental-dom",
   "version": "0.1.0",
   "description": "An in-place virtual DOM library",
-  "main": "index.js",
+  "main": "dist/incremental-dom.js",
   "author": "The Incremental DOM Authors",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/google/incremental-dom.git"
   },
+  "files" : [
+    "dist"
+  ],
   "scripts": {
-    "test": "gulp unit",
-    "lint": "gulp lint",
-    "build": "gulp build",
-    "dist": "gulp dist"
+    "test": "node_modules/gulp/bin/gulp.js unit",
```

Comment:
Shouldn't be a need to add the file paths, just `gulp` and npm will find the appropriate executable.